pkg/node/manager/cell.go

```diff
@@ -63,6 +63,8 @@ type NodeManager interface {
 	// StartNeighborRefresh spawns a controller which refreshes neighbor table
 	// by sending arping periodically.
 	StartNeighborRefresh(nh datapath.NodeNeighbors)
+
+	ReleaseRestoredIdentities()
```

Comment:
This function in this interface seems out of place and leaks details across layers. We already have code to handle allocate and release of the restored identities. Adding a new very specific piece of logic to handle this in the NodeManager interface doesn't seem ideal. IMO it's better to leverage the existing path, and if there are bugs in that path then let's squash them.

Reply:
Agreed. I can understand the motivation to prefer separate restoration logic that's specific to node CIDR identities and avoid touching the existing logic out of fear of dealing with refcounting bugs, but the proposed approach actually doesn't fundamentally change the fact that there _still_ could be a refcounting bug already existing in the code. 

A change that introduces the minimal diff to account for node CIDR identities means the chance of regression or a bug would be smaller than the current proposed changes, which involve a few layers.